js/app.js

```diff
@@ -17,6 +17,10 @@ function bdaySort(contact1, contact2) {
 }
 
 function bdayAge(contact) {
+  if (!(contact.bday instanceof Date)) {
```

Comment:
just out of curiosity, in what situation is this `contact.bday` not an instance of `Date` ?